example/App.js

```diff
@@ -5,7 +5,41 @@ import Dialog from '@material-ui/core/Dialog';
 import Paper from '@material-ui/core/Paper';
 import Switch from '@material-ui/core/Switch';
 import FormControlLabel from '@material-ui/core/FormControlLabel';
-import MultiModalComponent, { SearchConfig, searchTypes } from '../src/index';
+import gql from 'graphql-tag';
```

Comment:
what's the difference between this and https://github.com/trompamusic/trompa-multimodal-component-example ?

Reply:
The multimodal component example repository is a bit outdated. We are now using the example file in this repo. We were thinking about archiving the example repo.

Having the example in this repository makes things easier while developing the library.

Reply:
it's fine to have it included here, but if that's the case then I would definitely archive the other one, because I thought it was the primary example repo.

Reply:
Yes, I've archived the repo. The documentation and example in this repository should be leading. There are no references to that repository as far as we know.